cmd/aws-lambda-rie/main.go

```diff
@@ -62,12 +62,22 @@ func getBootstrap(args []string, opts options) (*rapidcore.Bootstrap, string) {
 	currentWorkingDir := "/var/task" // default value
 
 	if len(args) <= 1 {
-		bootstrapLookupCmd = []string{
+		var bootstrapCmdCandidates = []string{
 			fmt.Sprintf("%s/bootstrap", currentWorkingDir),
 			optBootstrap,
 			runtimeBootstrap,
 		}
 
+		// set default value to /var/task/bootstrap, but switch to the other options if it doesn't exist
+		bootstrapLookupCmd = []string{bootstrapCmdCandidates[0]}
```

Comment:
Probably minor, but it looks somewhat weird to set this default value, but then changing the variable to potentially this same value again if it passes the "if" below.

Reply:
Thanks, I have refactored this part to make it clearer.